pyproject.toml

```diff
@@ -1,25 +1,15 @@
 [tool.poetry]
 name = "adaptive_scheduler"
-version = "1.5.1"
+version = "1.5.2"
 description = ""
 authors = ["LCO <ocs@lco.global>"]
 license = "GPLv3"
 
-[[tool.poetry.source]]
-name = "ocs"
-url = "https://observatorycontrolsystem.github.io/pypi-simple/"
-
 [tool.poetry.dependencies]
-python = "^3.7,<3.11"
-ortools = [
-  { version = "9.3.9999", markers = "python_version >= '3.7' and python_version < '3.10' and platform_machine == 'x86_64'", source = "ocs" },
-  { version = "9.3.9999", markers = "python_version >= '3.7' and python_version < '3.10' and platform_machine == 'aarch64'", source = "ocs" },
-  { version = "9.3.10497", markers = "python_version >= '3.10'"},
-  { version = "9.3.10497", markers = "sys_platform != 'linux'"},
-  { version = "9.3.10497", markers = "sys_platform == 'linux' and (platform_machine != 'x86_64' and platform_machine != 'aarch64')"},
-]
+python = "^3.8,<3.12"
+ortools = "^9.10"
 time-intervals = "^1"
-numpy = "^1"
+numpy = "1.23.5"
```

Comment:
I think I had to fix this version because for some reason poetry was getting a higher version, but then complaining later that pyslalib needed this version (but for some reason it didn't figure that out when making the lock file).